source/custom/rules/cpuid_rule.go

```diff
@@ -29,6 +29,7 @@ type CpuIDRule struct {
 	nfdv1alpha1.MatchExpressionSet
 }
 
+// Match returns all matched keys
```

Comment:
nit: it doesn't return any keys

```suggestion
// Match checks if the CpuId rule matches.
```

Reply:
changed